### PR TITLE
Adding screenshots in README.md

### LINKED ISSUE #65: Add screenshots of app to the ReadMe
Currently Readme doesnot have screenshots of the app . It should be added to the ReadMe to give a clear idea of functionality of the app .
Would you like to work on this issue?
Yes

### PR DESCRIPTION
**Pull request by SIMRAN1**
Fixes #65 
Added screenshots in Readme.md.

### CODE REVIEW

**Comment by SIMRAN1** [comment]
@imjog I have added screenshots in README but build is failing!

**Review by jogendra — CHANGES_REQUESTED** [review]
- Solve the Option issue showing in screenshots
- Add screenshot in 2x2 table not 3x1

**Comment by SIMRAN1** [comment]
okk @imjog !!

**Comment by jogendra** [comment]
@SIMRAN1 Please check the resolved failing tests: #73

**Comment by ghost** [comment]
Hi @SIMRAN1!

It looks like one or more of your builds have failed.
I've added the relevant info below to save you some time.

- [Build 125.1](https://travis-ci.org/fossasia/open-event-ios/jobs/384573876) has `failed`

**Comment by SIMRAN1** [comment]
@imjog done the changes.Now screenshots are 9 so I think 3*3 would be better!

**Comment by imjog** [comment]
@SIMRAN1 Screenshot number **2, 5, 6 and 7** are unnecessary. They are copies or not need to add them to README. Purpose of Readme is to give an overview of an app, not the whole screens. Keep **4** screenshots, 4 is enough.
Do not give height or width to them in markdown, add them using `<table>`, they will adjust according to the screen (Last screenshot is squashing). Do not waste time on little things that are low priority, please work on the issues that make app better. thank you

**Review by jogendra — COMMENTED** [review]
@SIMRAN1 Screenshot number **2, 5, 6 and 7** are unnecessary. They are copies or not need to add them to README. Purpose of Readme is to give an overview of an app, not the whole screens. Keep **4** screenshots, 4 is enough.
Do not give height or width to them in markdown, add them using `<table>`, they will adjust according to the screen (Last screenshot is squashing). Do not waste time on little things that are low priority, please work on the issues that make app better. thank you

**Comment by SIMRAN1** [comment]
okk @imjog .Actually I was busy with my exams . I am making a plan to make the app better !!

**Comment by chashmeetsingh** [comment]
Try adding screenshots of any one device like iPhone X plus make sure the height of all the screenshots is the same.

**Comment by SIMRAN1** [comment]
okk @chashmeetsingh !!

**Comment by SIMRAN1** [comment]
@chashmeetsingh please review and merge it!!

File: README.md [hunk 1]
```diff
@@ -17,6 +17,19 @@
 
 > *https://gitter.im/fossasia/open-event*
 
+
+## Screenshots
+
+<table>
+<tr>
+<td><img src="docs/_static/images/event-list-details.png"  height = "480" ></td>
```
Comment (jogendra) [inline README.md:25]: Do not fix height, screenshot look squashed.

**Review by jogendra — CHANGES_REQUESTED** [review]
@SIMRAN1 Add two screen in a row, four look odd and squashed. Do not fix height, you can fix width.

File: README.md (hunk 1 above)
Comment (SIMRAN1) [reply README.md:25]: okk @imjog !!

**Comment by SIMRAN1** [comment]
@imjog @chashmeetsingh I am closing this PR for now . I will open it when new screens are designed.

**Pull request closed without merging**